### PR TITLE
fix: modified setup in Readme.md

### PR DESCRIPTION
**Pull request by vj-codes**
#### Is your pull request related to a problem? Please describe
Fixed issue #98

#### Checklist
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **[CONTRIBUTING](https://github.com/fbdevelopercircles/open-source-edu-bot/blob/master/CONTRIBUTING.md)** document.
- [ ] All new and existing tests passed.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,13 +10,29 @@ Open Source Education Bot built by the Facebook Developer Circles community to h
...
 ```bash
-git clone https://github.com/fbdevelopercircles/open-source-edu-bot
+git clone https://github.com/fbdevelopercircles/YOUR_GITHUB_USER_NAME/open-source-edu-bot 
+```
+
+where YOUR_GITHUB_USER_NAME is your GitHub handle.
+
+```
 cd open-source-edu-bot
+git remote add upstream https://github.com/fbdevelopercircles
```
Comment (hoduchieu01) [inline README.md:23]: ```git remote add upstream https://github.com/fbdevelopercircles``` is not working

It should be update as the following:

--
Firstly, you have to use git remote -v for managing set of tracked repositories.You have to add a new remote git remote add upstream https://github.com/fbdevelopercircles/open-source-edu-bot

File: README.md [hunk 2]
```diff
@@ -10,13 +10,29 @@ Open Source Education Bot built by the Facebook Developer Circles community to h
...
 ```
 
+You can check that the previous command worked by running git remote -v. You should see the following output:
+```
+git remote -v
+origin  https://github.com/YOUR_GITHUB_USER_NAME/fbdevelopercircles (fetch)
+origin  https://github.com/YOUR_GITHUB_USER_NAME/fbdevelopercircles (push)
+upstream        https://github.com/fbdevelopercircles (fetch)
```
Comment (hoduchieu01) [inline README.md:31]: Update this line

File: README.md [hunk 3]
```diff
@@ -10,13 +10,29 @@ Open Source Education Bot built by the Facebook Developer Circles community to h
...
+upstream        https://github.com/fbdevelopercircles (push)
```
Comment (hoduchieu01) [inline README.md:32]: Update this line

**Comment by vj-codes** [comment]
> You can follow Syncing a fork section in this guide to update 
> 
> https://cshandbook.hoduchieu.tech/GitHandbook/GitCommands.html

Yeah @hoduchieu01 
Only half part got copied from my git bash window. I'll edit it soon:)

**Comment by vj-codes** [comment]
@hoduchieu01 done!

**Comment by hoduchieu01** [comment]
@vj-codes Great work!

**Pull request merged**